pipeline_games/load_games.py

```diff
@@ -0,0 +1,289 @@
+                    variable=sql.Identifier(variable), table=sql.Identifier(table),
+                    value_name=sql.Identifier(value_name)), [value])
+            id_value = cur.fetchone()[variable]
+            cache[value] = id_value
+            cur.close()
+            return cache[value]
+    except TypeError:
+        return "None"
+
+
+def get_existing_data_for_genre(genre_c: str, user_c: bool, conn: connect, cache: dict) -> int | None:
+    """Retrieves the existing data for genre using cache and return genre_id"""
+    value = f'{genre_c} {user_c}'
+    if value in cache.keys():
+        return cache[value]
+    try:
+        with conn.cursor(cursor_factory=RealDictCursor) as cur:
+            cur.execute("""SELECT genre_id FROM genre WHERE genre = %s AND user_generated = %s;""", [
+                        genre_c, user_c])
+            id_value = cur.fetchone()['genre_id']
+            cache[value] = id_value
+            cur.close()
+            return cache[value]
+    except TypeError:
+        return "None"
+
+
+def get_existing_platform_data(mac_c, windows_c, linux_c, conn: connect, cache: dict) -> int | None:
+    """Retrieves the existing data for platform using cache and return platform_id"""
+    value = f'{mac_c} {windows_c} {linux_c}'
+    if value in cache.keys():
+        return cache[value]
+
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        cur.execute("""SELECT platform_id FROM platform
+                    WHERE mac = %s AND windows = %s AND linux = %s;""",
+                    [mac_c, windows_c, linux_c])
+        id_value = cur.fetchone()['platform_id']
+        cache[value] = id_value
+        cur.close()
+        return cache[value]
+
+
+def add_to_genre_link_table(conn: connect, data: list) -> None:
+    """Updates genre link table"""
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        cur.execute(
+            """SELECT genre_id FROM genre WHERE genre = %s AND user_generated = %s;""",
+            [data[12], data[13]])
+        genre_id = cur.fetchone()['genre_id']
+        cur.execute(
+            """SELECT game_id FROM game WHERE app_id = %s;""",
+            [data[2]])
+        game_id = cur.fetchone()['game_id']
+        cur.execute(
+            """SELECT exists (SELECT 1 FROM game_genre_link 
+            WHERE game_id = %s AND genre_id = %s LIMIT 1);""",
+            [game_id, genre_id])
+        result = cur.fetchone()
+        if result['exists'] is True:
+            cur.close()
+        elif result['exists'] is False:
+            cur.execute("""INSERT INTO game_genre_link(game_id, genre_id)
+                        VALUES (%s, %s);""", [game_id, genre_id])
+        conn.commit()
+        cur.close()
+
+
+def add_to_publisher_link_table(conn: connect, data: list) -> None:
+    """Updates publisher link table"""
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
```

Comment:
did you know you don't need to specify `cursor_factory=RealDictCursor` every time if you're always using it but can you add this as one of the lines in the connect() function from psycopg2. So to the line 18 in the function get_db_connection